### PR TITLE
♻️ enhances emails plugin and 🐛 fixes missing fields ( ⚠️ devops ) #3822

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Follows up on #3822: emails sent by osparc are classified as spam because some fields are missing. This PR adds missing MIME fields and creates a new entrypoint to test emails directly in the deploy (admin privileges needed). This should help spotting other issues that might be missing in the email server configured in the stack.

## ⚠️ devops:
There should be *one* (and one only) `admin` account per deployment (for now, best with mail our common devops mail adress osparcio-devops =at= speag.swiss). No other user (even team members) should have admin privileges. The admin-account will go into the e2e-ops, into the credentials page. This needs to be done upon releasing this PR.

### Hightlights 
- ♻️ Moves email and text templates helpers to ``email`` plugin
    - New ``POST /email:test`` for ``admin`` to manually test templates and email server (i.e. admin privileges required)
-  🐛 Completes required MIME date field 

## Related issue/s

- closes #3822

## How to test

- New entrypoint was designed to manually test with the actual email server
image

## Checklist

- [x] ``make version-*``
- [x] ``make openapi.json``
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes

### CODE REVIEW

**Review by mrnicegyu11 — APPROVED** [review]
Thanks a lot, truly <3 !

File: services/web/server/src/simcore_service_webserver/email_core.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            await smtp.send_message(message)
+
+
+def _compose_mime(
+    message: Union[MIMEText, MIMEMultipart],
+    *,
+    sender: str,
+    recipient: str,
+    subject: str,
+) -> None:
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/email_core.py:83]: type is incorrect

Comment (GitHK) [reply services/web/server/src/simcore_service_webserver/email_core.py:83]: maybe you do not want to return here?

File: services/web/server/src/simcore_service_webserver/email_handlers.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+logger = logging.getLogger(__name__)
+
+
+#
+# API schema models
+#
+
+
+class TestEmail(BaseModel):
+    from_: Optional[EmailStr] = Field(None, description="Email sender")
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/email_handlers.py:29]: why is the field called `from_`?

**Review by GitHK — APPROVED** [review]
👍

File: services/web/server/src/simcore_service_webserver/email_handlers.py (hunk 2 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/email_handlers.py:29]: ``from`` is a reserved word e.g. ``from pydantic import *``

**Pull request merged**